torch_geometric/explain/explainer.py

```diff
@@ -46,7 +44,8 @@ def __init__(
         self.algorithm.connect(self.explainer_config, self.model_config)
 
     @torch.no_grad()
-    def get_prediction(self, *args, **kwargs) -> torch.Tensor:
+    def get_prediction(self, *args, raw: bool = False,
+                       **kwargs) -> torch.Tensor:
```

Comment:
Can you clarify why we need this? If we want to return raw predictions, we should apply the `argmax` in the loss function.

Reply:
The `get_prediction` method was already in the `Explainer` class, but it applied `argmax` by default. In explanations we might want to compare the difference in probability when using only the explanation graph as input, hence the `raw` flag. 

I am fine with moving `argmax` into the loss function and returning the raw value by default. Wdyt?